### PR TITLE
Aligning config and lang en values to match a standard.

### PR DESCRIPTION
**Pull request by DerekRushton**

### CODE REVIEW

File: stix_shifter_modules/aws_athena/configuration/config.json [hunk 1]
```diff
@@ -49,8 +49,7 @@
         "auth": {
             "type" : "fields",
             "aws_access_key_id": {
-                "type": "password",
-                "optional": true
```
[the remark this answers is not in the document]
Comment (DerekRushton) [reply stix_shifter_modules/aws_athena/configuration/config.json:53]: This one I wasn't sure on. The problem is that the other Amazon connectors (which share these same fields) do not have the field as optional.

I was conflicted between making them all optional (across all the Amazon's) or changing the key/secret in Athena to match Guardduty/Cloudwatch.

Realistically, this one is probably more like an optional as it should be role-based OR access_key based authentication, but there isn't an option for XOR here.

File: stix_shifter_modules/azure_log_analytics/configuration/lang_en.json [hunk 2]
```diff
@@ -1,21 +1,21 @@
 {
   "connection": {
     "host": {
-      "label": "Management IP address or Hostname",
-      "description": "Specify the IP address or hostname of the data source"
+      "label": "Management IP address or hostname",
+      "description": "Specify the IP address or  hostname of the data source so that IBM Cloud Pak for Security can communicate with it"
```
Comment (mdazam1942) [inline stix_shifter_modules/azure_log_analytics/configuration/lang_en.json:5]: you need to remove the mention of `IBM Cloud Pak for Security` from the description. Maybe just revert back the change. This applies to all `host` description changes in this pr

Comment (DerekRushton) [reply stix_shifter_modules/azure_log_analytics/configuration/lang_en.json]: I changed all of the host to use 
`Specify the IP address or hostname of the data source`

File: stix_shifter_modules/aws_athena/configuration/config.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (mdazam1942) [reply stix_shifter_modules/aws_athena/configuration/config.json:53]: lets make them optional across all the amazon connectors

File: stix_shifter_modules/rhacs/configuration/lang_en.json [hunk 3]
```diff
@@ -13,15 +13,15 @@
...
-    }
+      "label": "PEM Formatted SSL certificate(s)",
+      "description": "Provide a self-signed or CA-signed certificate to securely communicate with the data source."
+  }
   },
   "configuration": {
     "auth": {
       "token": {
-        "label": "API token",
-        "description": "Token with readonly access to the alert API"
```
Comment (mdazam1942) [inline stix_shifter_modules/rhacs/configuration/lang_en.json:24]: we need to include this piece of information about `readonly access to the alert API` in the description

File: stix_shifter_modules/proofpoint/configuration/lang_en.json [hunk 4]
```diff
@@ -16,13 +15,13 @@
     },
     "configuration": {
         "auth": {
-            "principal": {
-                "label": "Principal",
-                "description": "Principal with access to the SIEM API"
+            "principle": {
+                "label": "Principle",
+                "description": "Principle required for data source authentication"
```
Comment (mdazam1942) [inline stix_shifter_modules/proofpoint/configuration/lang_en.json:20]: keep the `SIEM API ` in the descirption

File: stix_shifter_modules/gcp_chronicle/configuration/lang_en.json [hunk 5]
```diff
@@ -12,12 +12,12 @@
...
             "client_email": {
-                "label": "Client Email",
-                "description": "Client Email used in authentication to make API calls"
+                "label": "Client email address",
+                "description": "Client email address required for data source authentication"
             },
             "private_key": {
-                "label": "Private Key",
-                "description": "Private Key is a authentication parameter used to communicate with the GCP Chronicle security datasource."
+                "label": "Private Key (Required)",
```
Comment (mdazam1942) [inline stix_shifter_modules/gcp_chronicle/configuration/lang_en.json:19]: do we need `(Required)` in the label?

**Review by mdazam1942 — APPROVED** [review]

**Pull request merged**